edflow/explore.py

```diff
@@ -63,6 +66,11 @@ def display(key, obj):
     elif sel == "Flow":
         display_flow(obj, key)
 
+    elif sel == "Segmentation":
```

Comment:
Nice! That's really good to have!
Two suggestions for the future (not this PR): 
1. It might be beneficial to move the conversion code outside of edexplore, so that we can also use it to store segmentation maps e.g. with the eval pipeline or for standart training logging.
2. To make sure the users understand they are looking at  a segmentation, we might want to use a colormap like viridis or something even more varied.